pkg/logs/tailers/windowsevent/tailer.go

```diff
@@ -46,6 +46,46 @@ const (
 type Config struct {
 	ChannelPath string
 	Query       string
+	// V1ProcessingBehavior indicates ifq we want to process and send the whole structured log message
+	// instead of on the logs content. Note that the default behavior is now to only send
+	// the logs content, as other tailers do.
+	V1ProcessingBehavior bool
+}
+
+// eventContext links go and c
+type eventContext struct {
+	id int
+}
+
+// WindowsEventMessage is used by the tailer to store the structured log information.
+// The message from the log is in the "message" key.
+type WindowsEventMessage struct { //nolint:revive
+	data mxj.Map
+}
+
+// Render renders the structured log information into JSON, for further encoding before
+// being sent to the intake.
+func (m *WindowsEventMessage) Render() ([]byte, error) {
+	data, err := m.data.Json(false)
+	if err != nil {
+		return nil, err
+	}
+	log.Trace("Rendered JSON in structured message:", string(jsonEvent))
+	return string(replaceTextKeyToValue(data)), nil
+}
+
+// GetContent returns the content part of the structured log.
+func (m *WindowsEventMessage) GetContent() []byte {
+	if message, exists := m.data["message"]; exists {
+		return message.([]byte)
+	}
+	log.Error("WindowsEventMessage not containing any message")
+	return []byte{}
```

Comment:
I am not 100% sure that absence of message is abnormal. @clarkb7 should have more context